test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -222,15 +268,21 @@ private static string GetBuildArgs(params string[] args)
             return buildArgs;
         }
 
-        public static string GetDotNetImage(string imageVersion, DotNetImageType imageType, string osVariant)
+        public static string GetDotNetImage(string imageVersion, DotNetImageType imageType, string osVariant, string architecture)
         {
             string variantName = Enum.GetName(typeof(DotNetImageType), imageType).ToLowerInvariant().Replace('_', '-');
             string imageName = $"microsoft/dotnet-nightly:{imageVersion}-{variantName}";
+
             if (!string.IsNullOrEmpty(osVariant))
             {
                 imageName += $"-{osVariant}";
             }
 
+            if (String.Equals("ARM", architecture, StringComparison.OrdinalIgnoreCase))
+            {
+                imageName = $"microsoft/dotnet-nightly:{imageVersion}-runtime-stretch-arm32v7";
```

Comment:
There is too much hard coded here which will make it difficult to handle between repo's and as the os evolves.  It seems like the architecture is all that is needed - `imageName += $"-arm32v7";`  The caller should be specifying `stretch` as the osVariant.

Reply:
Yes, I'll minimize the hard-coding. I see there is `microsoft/dotnet-nightly:2.0-runtime-stretch-arm32v7` but no ``microsoft/dotnet-nightly:2.0-runtime-deps-stretch-arm32v7``

Reply:
Resolved based on offline discussion.